src/test/java/APITest/List_users.java

```diff
@@ -0,0 +1,64 @@
+package APITest;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+import ResponsePojo.Response_single_user;
+
+public class List_users {
+
+public static void main(String[] args) {
+	List_users.All_list_users();
+	List_users.Delete_user();
+	List_users.list_single_user();
+	List_users.single_use_notfound();
+	List_users.List_resources();
+	List_users.List_single_resources();
+	List_users.single_resource_notfound();
+}
+		        
+public static void All_list_users() {
+	given().spec(Utility.requestbuild()).queryParam("page", "2")
+	.when().get("/api/users")
+	.then().spec(Utility.responsebuild())
+	.body("data.id", hasItems(7,8));    
+  }      
+
+public static void Delete_user() {
```

Comment:
File name should be `camelCased`. Please read about naming conventions here: https://www.geeksforgeeks.org/java-naming-conventions/